app/shared-components/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -23,8 +23,15 @@ import ClassNames from 'classnames';
 const ToggleSwitch = (props) => {
 	const switchClassNames = ClassNames('ToggleSwitch', {
 		'ToggleSwitch--active': props.checked,
+		'ToggleSwitch--locked': props.locked,
+		'ToggleSwitch--removed': props.removed
 	});
 
+	ToggleSwitch.defaultProps = {
+		locked: false,
+		removed: false
+	};
+
```

Comment:
We add the `.defaultProps` at the bottom of the file, separate from the class or function declaration. See other `.defaultProps` declarations as guidance.

Reply:
Very neat. I have moved it to the bottom of the file.